pkg/util/scrubber/default_test.go

```diff
@@ -493,6 +493,16 @@ func TestAddStrippedKeys(t *testing.T) {
 	assertClean(t, contents, `foobar: "********"`)
 }
 
+func TestAddStrippedKeysAPIKey(t *testing.T) {
+	contents := `api_key: 'aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa'`
+
+	AddStrippedKeys([]string{"api_key"})
```

Comment:
Could we test adding more than 1 string to test that the filtering clean just what we want and nothing more. Also, we should test for `app_key`